html/pfappserver/lib/pfappserver/PacketFence/Controller/Graph.pm

```diff
@@ -725,6 +728,12 @@ sub connectiontype :Local :AdminRole('REPORTS') {
                                    'count' => 'connections' },
                      }
                     );
+    my $items = $c->stash->{items};
+    if ($items) {
+        for my $item (@$items) {
+            $item->{link} = '/admin/nodes#/node/advanced_search?searches.0.name=connection_type&searches.0.op=equal&searches.0.value=' . $pf::config::connection_type_explained_to_str{$item->{label}};
```

Comment:
To reuse `_add_links`, could you define a new attribute to `$item` and use it as the `$value_key` argument?